packages/datadog-plugin-fetch/test/integration-test/client.spec.js

```diff
@@ -0,0 +1,51 @@
+'use strict'
+
+const {
+  FakeAgent,
+  createSandbox,
+  curlAndAssertMessage,
+  skipUnsupportedNodeVersions,
+  spawnPluginIntegrationTestProc
+} = require('../../../../integration-tests/helpers')
+const { assert } = require('chai')
+
+const describe = (globalThis.fetch && skipUnsupportedNodeVersions() === globalThis.describe)
+  ? globalThis.describe : globalThis.describe.skip
```

Comment:
I think these two lines can be simplified. Basically, the code here is re-implementing the logic in `skipUnsupportedNodeVersions`. Maybe this:

```js
const describe = globalThis.fetch ? skipUnsupportedNodeVersions() : globalThis.describe.skip
```